esf_main.tex

```diff
@@ -13,14 +13,13 @@
 \subsection{Ready-To-Drive-Sound (RTDS)}
 \subsubsection{Description}
 %Describe your concept of the RTDS, how is the sound produced, what are the parameters for activating the RTDS, etc.
-The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.puiaudio.com/pdf/XL-5530L-LW300-S-R.pdf}{Link: PUI Audio Inc. XL-5530L-LW300-S-R}). The buzzer automatically makes a noise when given power, with the loudness proportional
-to the voltage. The last step in the start up sequence will notify the CAN system that it is time for the ready
-to drive sound. This will then provide voltage to the gate of an N-FET which will provide 12V to the buzzer.
+The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.mallory-sonalert.com/specifications/STA20502.PDF}{Link: Mallory Sonalert Products Inc. STA20502}). The buzzer automatically makes a noise when given power, with the loudness proportional
```

Comment:
Please provide a reference with \ref to a subsection of the appendix that shows a piece of the data sheet with volume charcteristics, and also has a hyperlink to the full datasheet, as per last year's ESF.